tasks/setup/vss-extension.json

```diff
@@ -0,0 +1,41 @@
+{
+  "manifestVersion": 1,
+  "id": "setup-infracost-task",
+  "name": "Setup Infracost",
+  "version": "0.0.15",
+  "publisher": "infracost",
+  "targets": [
+    {
+      "id": "Microsoft.VisualStudio.Services"
+    }
+  ],
+  "description": "Sets up Infracost CLI in your Azure Devops pipeline. Show cloud cost estimates for Terraform in pull requests.",
+  "categories": [
+    "Azure Pipelines"
+  ],
+  "icons": {
+    "default": "images/extension-icon.png"
+  },
+  "files": [
+    {
+      "path": "InfracostSetup"
+    }
+  ],
+  "content": {
+    "details": {
+      "path": "overview.md"
+    }
+  },
+  "contributions": [
+    {
+      "id": "infracost-setuptask",
+      "type": "ms.vss-distributed-task.task",
+      "targets": [
+        "ms.vss-distributed-task.tasks"
+      ],
+      "properties": {
+        "name": "InfracostSetup"
+      }
+    }
+  ]
+}
```

Comment:
Please add a newline character.

Reply:
done in https://github.com/infracost/infracost-azure-devops/pull/9/commits/95030bf9394c362cb56a6bdec90109b014fcd819